### ISSUE TITLE
Add className support

### ISSUE THREAD

**Issue by devcisne** [frontend-joe/react-bubbly-transitions#1]
Hello, recently seen your post on instagram and was thinking of integrating this into my personal site. However, I encountered a problem as I am using tailwindcss to style my `<Link>` components and when replacing them for `<BubblyLink>` realised it does not support adding classes via className.

**Comment by frontend-joe** [comment]
Hey dude, would me adding a className prop just solve this issue?

Cheers

Joe

Sent from my iPhone

On 27 Oct 2022, at 10:58, Diego Sebastian Cisneros Vitor ***@***.***> wrote:

﻿

Hello, recently seen your post on instagram and was thinking of integrating this into my personal site. However, I encountered a problem as I am using tailwindcss to style my components and when replacing them for realised it does not support adding classes via className.

**Comment by devcisne** [comment]
thanks for your quick reply and work on these cool components, I've found this [stackOverflow link](https://stackoverflow.com/questions/54377517/add-additonal-class-names-to-custom-react-component) which is just what it's needed  i think.

**Comment by frontend-joe** [comment]
Yeah you could, or just create it it’s own style sheet and inject the tailwind classes via css

[image0.png]

Sent from my iPhone

On 27 Oct 2022, at 11:09, Diego Sebastian Cisneros Vitor ***@***.***> wrote:

﻿

thanks for your quick reply and work on this cool components, I've found this stackOverflow link<https://stackoverflow.com/questions/54377517/add-additonal-class-names-to-custom-react-component> which is just what it's needed i think.

**Comment by devcisne** [comment]
the image [image0.png] is not visible
